Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN apt-get update && apt-get -qy install --no-install-recommends \
  python3.8 \
  python3-pip \
  libmysqlclient-dev \
+ pkg-config \
```

Comment:
Don't forget to add a comment describing what pkg-config is for above the `RUN` line (see comment on line 23).